testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/rest/TestingResourceProvider.java

```diff
@@ -192,6 +208,280 @@ public Response clearAdminEventQueue() {
     }
 
     @GET
+    @Path("/clear-event-store")
```

Comment:
We already have support for clearing events in the admin endpoints. Why add it here as well?